third_party/istio-1.2.0/values-extras.yaml

```diff
@@ -0,0 +1,86 @@
+# Common settings.
+global:
+  omitSidecarInjectorConfigMap: true
+  istioNamespace: istio-system
+  proxy:
+    # Enable proxy to write access log to /dev/stdout.
+    accessLogFile: "/dev/stdout"
+    autoInject: disabled
+  disablePolicyChecks: true
+
+
+gateways:
+  enabled: true
+
+  istio-ingressgateway:
+    enabled: false
+  istio-egressgateway:
+    enabled: false
+  istio-ilbgateway:
+    enabled: false
+
+  cluster-local-gateway:
+    enabled: true
+    labels:
+      app: cluster-local-gateway
+      istio: cluster-local-gateway
+    replicaCount: 1
+    autoscaleMin: 1
+    autoscaleMax: 1
+    resources: {}
+    cpu:
+      targetAverageUtilization: 80
+    loadBalancerIP: ""
+    loadBalancerSourceRanges: {}
+    externalIPs: []
+    serviceAnnotations: {}
+    podAnnotations: {}
+    type: ClusterIP
+    ports:
+    - name: status-port
+      port: 15020
+    - name: http2
+      port: 80
+    - name: https
+      port: 443
+    secretVolumes:
+    - name: clusterlocalgateway-certs
+      secretName: istio-clusterlocalgateway-certs
+      mountPath: /etc/istio/clusterlocalgateway-certs
+    - name: clusterlocalgateway-ca-certs
+      secretName: istio-clusterlocalgateway-ca-certs
+      mountPath: /etc/istio/clusterlocalgateway-ca-certs
+
+# all other components are disabled except the gateways
```

Comment:
@markusthoemmes  Updated the values-extra.yaml to keep same value with value-lean.yaml and value.yaml.
However, from line 54 to end of this file, do I need to remove these lines or just keep here? This is copied from sample values of istio installation.